cvo-overrides-after-first-run.yaml

```diff
@@ -3,54 +3,91 @@
   value:
   - kind: Deployment
     group: apps/v1
-    name: machine-config-operator
-    namespace: openshift-machine-config-operator
+    name: cluster-monitoring-operator
+    namespace: openshift-monitoring
     unmanaged: true
-  - kind: Deployment
-    group: apps/v1
-    name: etcd-quorum-guard
-    namespace: openshift-machine-config-operator
+  - kind: ClusterOperator
```

Comment:
@guillaumerose do we need those `ClusterOperators` as `kind` in the override list also, doesn't setting up the deployment as unmanagement  works?

Reply:
yes it is needed as the installer is waiting for all cluster operators to be ready.